jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -78,6 +80,7 @@ class java_bytecode_parsert:public parsert
   };
 
   std::vector<bytecodet> bytecodes;
+  bool skip_instructions;
```

Comment:
:pick: 
```suggestion
  bool skip_instructions = false;
```
if someone adds a new constructor and forgets to initialise this field we won't be left in UB